app/views/org_admin/questions/_show.html.erb

```diff
@@ -58,6 +58,36 @@
           <% end %>
         </dl>
       </div>
+      <div class="col-md-4">
+        <% if guidance.present? %>
+          <% themes_q = question.themes %>
+          <% if !themes_q.nil? %>
+            <div class="accordion" id="<%= question.id %>-guidance">
+            <% themes_q.each do |theme| %>
+              <% theme.guidances.each do |guidance| %>
+                <% GuidanceGroup.where(org_id: current_user.org_id).each do |guidance_group| %>
```

Comment:
I would switch this up to get the guidance groups first and then use the guidance group ids to filter out the guidance you want on the DB side. (See code snippet in my final notes)